rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -15,6 +15,7 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 [%header,cols="10,20,30,10"]
 |===
 |Icon |Name | Description | Hotkey
+| |Image-based mouse clicks | Add recording with the <<Image-Based Mouse Clicks>> feature. | `F3`
```

Comment:
This is missing the icon. Also, the description is not parallel to the other table descriptions.

Reply:
Thank you! The icon was pending availability of the latest RPA Recorder version. I'll be adding that shortly.